main/dmenu/APKBUILD

```diff
@@ -1,26 +1,25 @@
 # Contributor: Eivind Uggedal <eivind@uggedal.com>
 # Maintainer: Sören Tempel <soeren+alpine@soeren-tempel.net>
 pkgname=dmenu
-pkgver=4.7
+pkgver=4.8
 pkgrel=0
 pkgdesc="Dynamic menu for X"
-url="https://tools.suckless.org/dmenu/"
+url="https://tools.suckless.org/dmenu"
 arch="all"
 license="MIT"
 depends=""
 depends_dev=""
 makedepends="libx11-dev libxinerama-dev libxft-dev"
 install=""
-options="!check"
+options="!check" # no upstream/available testsuite
 subpackages="$pkgname-doc"
 source="http://dl.suckless.org/tools/$pkgname-$pkgver.tar.gz"
-builddir="$srcdir"/$pkgname-$pkgver
+builddir="$srcdir/$pkgname-$pkgver"
 
 prepare() {
-	default_prepare
```

Comment:
Why did you remove `default_prepare`? This should be always called in custom `prepare` function.